### ISSUE TITLE
Show current branch in bottom panel?

### ISSUE THREAD

**Issue by thybzi** [zaggino/brackets-git#1081]
Really great extension.
There is only one thing making in inconvinient to use.
Current branch is only displayed in sidebar, where there is not enough space to show it full (esp. if project name and/or branch name is long)

I suggest to make an option to display and switch current branch (also) in bottom `Git` panel, somewhere near `origin`.

(BTW, most users don't switch remotes too frequent, so maybe there also should be another option to hide `origin` from there).

Thank you in advance

--------------------------------

*Brackets 1.3.0-16022 (win), Git 1.9.2.msysgit.0 & Brackets-Git 0.14.27*

**Comment by joseporto** [comment]
Agree, I would like to see this changes. The branch should be on the bottom task bar.
